spec/suites/layer/GeoJSONSpec.js

```diff
@@ -512,3 +512,387 @@ describe("L.LayerGroup#toGeoJSON", function () {
+				[[0, 0], [1, 1], [2, 2]],
+				[[4, 4], [5, 5], [6, 6]]
+			], 1);
+			expect(latLngs.length).to.be(2);
+			for (var i = 0; i < latLngs.length; i++) {
+				for (var j = 0; j < latLngs[i].length; j++) {
+					expect(latLngs[i][j] instanceof L.LatLng).to.be(true);
+				}
+			}
+		});
+
+		it("creates a multidimensional array of LatLngs with custom coordsToLatLng", function () {
+			const coords = [[1, 2, 3], [4, 5, 6], [7, 8, 9]];
+			const latLngs = L.GeoJSON.coordsToLatLngs(coords, 0, customCoordsToLatLng);
+			const expected = [
+				{lat: 2, lng: 3, alt: 4},
+				{lat: 5, lng: 6, alt: 7},
+				{lat: 8, lng: 9, alt: 10}
+			];
+			for (var i = 0; i < latLngs.length; i++) {
+				expect(latLngs[i]).to.eql(expected[i]);
+			}
+		});
+
+		it("creates a multidimensional array of LatLngs with custom coordsToLatLng (levelDeep=1)", function () {
+			const coords = [
+				[[1, 2, 3], [4, 5, 6]],
+				[[12, 13, 14], [15, 16, 17]]
+			];
+			const latLngs = L.GeoJSON.coordsToLatLngs(coords, 1, customCoordsToLatLng);
+			const expected = [
+				[
+					{lat: 2, lng: 3, alt: 4},
+					{lat: 5, lng: 6, alt: 7}
+				],
+				[
+					{lat: 13, lng: 14, alt: 15},
+					{lat: 16, lng: 17, alt: 18}
+				]
+			];
+			for (var i = 0; i < latLngs.length; i++) {
+				for (var j = 0; j < latLngs[i].length; j++) {
+					expect(latLngs[i][j]).to.eql(expected[i][j]);
+				}
+			}
+		});
+	});
+
+	describe("#latLngToCoords", function () {
+		it("returns an array of coordinates (longitude, latitude)", function () {
+			const coords = L.GeoJSON.latLngToCoords(new L.LatLng(0, 0));
+			expect(coords).to.eql([0, 0]);
+		});
+
+		it("returns an array of coordinates and altitude (longitude, latitude, altitude)", function () {
+			const coords = L.GeoJSON.latLngToCoords(new L.LatLng(0, 0, 0));
+			expect(coords).to.eql([0, 0, 0]);
+		});
+
+		it("returns an array of coordinates with given precision (longitude, latitude)", function () {
+			const coords = L.GeoJSON.latLngToCoords(new L.LatLng(
+				1.123456, 1.123456
+			), 3);
+			expect(coords).to.eql([1.123, 1.123]);
+		});
+
+		it("returns an array of coordinates with given precision (longitude, latitude, altitude)", function () {
+			const coords = L.GeoJSON.latLngToCoords(new L.LatLng(
+				1.123456, 1.123456, 1.123456
+			), 3);
+			expect(coords).to.eql([1.123, 1.123, 1.123]);
+		});
+	});
+
+	describe("#latLngsToCoords", function () {
+		it("returns a multidimensional array of coordinates", function () {
+			const coords = L.GeoJSON.latLngsToCoords([L.latLng(0, 0), L.latLng(1, 1)]);
+			expect(coords).to.eql([[0, 0], [1, 1]]);
+		});
+
+		it("returns a multidimensional array of coordinates (levelDeep=1)", function () {
+			const latLngs = [
+				[L.latLng(0, 0), L.latLng(1, 1)],
+				[L.latLng(2, 2), L.latLng(3, 3)]
+			];
+			const coords = L.GeoJSON.latLngsToCoords(latLngs, 1);
+			const expected = [
+				[[0, 0], [1, 1]],
+				[[2, 2], [3, 3]]
+			];
+			expect(coords).to.eql(expected);
```

Comment:
May be matter of taste, but I feel this more expressive
```suggestion
			expect(coords).to.eql([
				[[0, 0], [1, 1]],
				[[2, 2], [3, 3]]
			]);
```

There are many such places in your code, so I will not comment each of them.